src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.cs

```diff
@@ -815,6 +815,438 @@ public new abstract class Arm64 : AdvSimd.Arm64
         public static unsafe Vector<ulong> Compact(Vector<ulong> mask, Vector<ulong> value) => Compact(mask, value);
 
 
+        ///  Conditionally extract element after last
+
+        /// <summary>
+        /// svuint8_t svclasta[_u8](svbool_t pg, svuint8_t fallback, svuint8_t data)
+        ///   CLASTA Ztied.B, Pg, Ztied.B, Zdata.B
+        /// </summary>
+        public static unsafe Vector<byte> ConditionalExtractAfterLastActiveElement(Vector<byte> mask, Vector<byte> defaultValue, Vector<byte> data) => ConditionalExtractAfterLastActiveElement(mask, defaultValue, data);
+
+        /// <summary>
+        /// svuint8_t svclasta[_u8](svbool_t pg, svuint8_t fallback, svuint8_t data)
+        ///   CLASTA Ztied.B, Pg, Ztied.B, Zdata.B
+        /// uint8_t svclasta[_n_u8](svbool_t pg, uint8_t fallback, svuint8_t data)
+        ///   CLASTA Wtied, Pg, Wtied, Zdata.B
+        ///   CLASTA Btied, Pg, Btied, Zdata.B
+        /// </summary>
+        public static unsafe byte ConditionalExtractAfterLastActiveElement(Vector<byte> mask, byte defaultValues, Vector<byte> data) => ConditionalExtractAfterLastActiveElement(mask, defaultValues, data);
+
+        /// <summary>
+        /// svfloat64_t svclasta[_f64](svbool_t pg, svfloat64_t fallback, svfloat64_t data)
+        ///   CLASTA Ztied.D, Pg, Ztied.D, Zdata.D
+        /// </summary>
+        public static unsafe Vector<double> ConditionalExtractAfterLastActiveElement(Vector<double> mask, Vector<double> defaultValue, Vector<double> data) => ConditionalExtractAfterLastActiveElement(mask, defaultValue, data);
+
+        /// <summary>
+        /// svfloat64_t svclasta[_f64](svbool_t pg, svfloat64_t fallback, svfloat64_t data)
+        ///   CLASTA Ztied.D, Pg, Ztied.D, Zdata.D
+        /// float64_t svclasta[_n_f64](svbool_t pg, float64_t fallback, svfloat64_t data)
+        ///   CLASTA Xtied, Pg, Xtied, Zdata.D
+        ///   CLASTA Dtied, Pg, Dtied, Zdata.D
+        /// </summary>
+        public static unsafe double ConditionalExtractAfterLastActiveElement(Vector<double> mask, double defaultValues, Vector<double> data) => ConditionalExtractAfterLastActiveElement(mask, defaultValues, data);
+
+        /// <summary>
+        /// svint16_t svclasta[_s16](svbool_t pg, svint16_t fallback, svint16_t data)
+        ///   CLASTA Ztied.H, Pg, Ztied.H, Zdata.H
+        /// </summary>
+        public static unsafe Vector<short> ConditionalExtractAfterLastActiveElement(Vector<short> mask, Vector<short> defaultValue, Vector<short> data) => ConditionalExtractAfterLastActiveElement(mask, defaultValue, data);
+
+        /// <summary>
+        /// svint16_t svclasta[_s16](svbool_t pg, svint16_t fallback, svint16_t data)
+        ///   CLASTA Ztied.H, Pg, Ztied.H, Zdata.H
```

Comment:
This instruction maps to the API that takes `Vector<T>` for `defaultValue` parameter. Please remove it from here and at other places.

Reply:
Done.